packages/material-ui/src/RootRef/RootRef.d.ts

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
 export interface RootRefProps {
-  rootRef?: () => React.HTMLAttributes<any>;
```

Comment:
The new API supports `RefObject`s _in addition to_ a callback function, right? So it seems like it should be something like

```ts
export interface RootRefProps<T = any> {
  rootRef?: ((instance: T | null) => void) | React.RefObject<T>;
}
```